src/Common/Commands.Common.Authentication/Factories/AuthenticationFactory.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.Azure.Commands.Common.Authentication.Factories
 {
     public class AuthenticationFactory : IAuthenticationFactory
     {
-        public const string CommonAdTenant = "Common";
+        public const string CommonAdTenant = "Common", DefaultMSILoginUri = "http://localhost:50342/oauth2/token";
```

Comment:
Is the default really a localhost? Just curious, I don't entirely understand how MSI works.

Reply:
@maddieclayton Yes, the default is always localhost - it is hosted on a VM at the loopback address.